src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+
+@WebServlet("/home")
+public class HomeServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+    response.setContentType("text/html;");
+    request.getRequestDispatcher("webapp/index.html").forward(request, response); 
+    //I'm not sure how to forward it to an HTML page in a different directory
```

Comment:
Let's say we have https://www.google.com/foo/bar.html

request.getRequestDispatcher("webapp/index.html").forward(request, response); -> goes to  https://www.google.com/foo/webapp/index.html

request.getRequestDispatcher("/webapp/index.html").forward(request, response); -> goes to  https://www.google.com/webapp/index.html

You can never forward requests to a different domain, but "/" lets you forward relative to the root vs. relative the

Reply:
Thank you for the feedback! I'm wondering will "../webapp/index.html" work?